test/input_gen/genInput.py

```diff
@@ -351,16 +411,55 @@ def bn_tf(x, *, trainable=True, init_beta=gen_tensor, init_gamma=gen_tensor, axi
 
 
 def gen_test_case_conv(i_b, i_c, i_h, i_w, k_c, k_h, k_w, padding, stride, bias, base_name):
-    x=gen_input(base_name+"_conv2DLayer.in", [i_b, i_c, i_h, i_w])
-    kernel=gen_input(base_name+"_conv2DKernel.in", [k_c, i_c, k_h, k_w])
+    x=gen_input(base_name+"_conv2DLayer.in", [i_b, i_c, i_h, i_w], int_data=True)
+    kernel=gen_input(base_name+"_conv2DKernel.in", [k_c, i_c, k_h, k_w], int_data=True)
     with open(base_name+"_conv2DKernel.in", 'ab') as outfile:
         np.array(bias, dtype=np.float32).tofile(outfile)
 
-    golden_conv, golden_grad_input, golden_grad_kernel, golden_grad_bias=conv2d_tf(x, kernel, i_b, i_h, i_w, i_c, k_h, k_w, k_c, stride, padding, bias)
+    golden_conv, golden_grad_input, golden_grad_kernel, golden_grad_bias, \
+        golden_conv2, golden_grad_input2, golden_grad_kernel2, golden_grad_bias2 = \
+        conv2d_tf(x, kernel, i_b, i_h, i_w, i_c, k_h, k_w, k_c, stride, padding, bias)
     save(base_name+"_goldenConv2DResult.out", np.transpose(golden_conv,(0,3,1,2)))
     save(base_name+"_goldenInputGrad.out", np.transpose(golden_grad_input,(0,3,1,2)))
     save(base_name+"_goldenKernelGrad.out", np.transpose(golden_grad_kernel,(3,2,0,1)))
     save(base_name+"_goldenBiasGrad.out", golden_grad_bias)
+    save(base_name+"_goldenConv2DResult2.out", np.transpose(golden_conv2,(0,3,1,2)))
+    save(base_name+"_goldenInputGrad2.out", np.transpose(golden_grad_input2,(0,3,1,2)))
+    save(base_name+"_goldenKernelGrad2.out", np.transpose(golden_grad_kernel2,(3,2,0,1)))
+    save(base_name+"_goldenBiasGrad2.out", golden_grad_bias2)
+
+def gen_test_case_conv_int(base_name):
```

Comment:
as all the unittests now use int like values, rename this to `gen_test_case_conv_multi_iter`

Reply:
fixed. thanks.